### ISSUE TITLE
Uppercase first and last name when registering users

### ISSUE THREAD

**Issue by rbaltrusch** [rbaltrusch/desktop_shop#10]
Currently, the database stores lowercase user names "as is" and does not convert them to title case (uppercase first letter of names) before storing them in the database.

Picture of status quo:
![python_9TlAWoxZKL](https://user-images.githubusercontent.com/41642326/144753840-3ef6e889-219b-41a7-9f7c-e1c222fb2f12.png)

The new functionality should be implemented inside the ```add_user``` and ```update_user``` functions in __database.py__.

**Comment by cnonnet** [comment]
Hello, I should be able to do this, could you assign me ?

**Comment by rbaltrusch** [comment]
Hi @cnonnet, sure 👍 Let me know if there are any issues.

**Comment by cnonnet** [comment]
Alright, I think is is working :)
I just manually changed the user_data to have an uppercase in the first_name and last name in the add_user function and on the update_user_by_user_email and update_user. I also modified the edit_user_data in callbacks.py otherwise the information on the user interface were not updated even though the information was different in the database.
I created a pull request with my modifications, I checked on my computer and it is working (see captures).
![register](https://user-images.githubusercontent.com/75025185/144768154-26196d7b-3b1e-4244-9282-b92d3abde9d7.JPG)
![result](https://user-images.githubusercontent.com/75025185/144768155-0bcb6b79-4044-4063-95d1-6f813e75f2c2.JPG)
s
